simple-git-hooks.test.js

```diff
@@ -275,7 +292,77 @@ test.each([
 
     spc.setHooksFromConfig(projectWithCustomConfigurationFilePath, args)
     const installedHooks = getInstalledGitHooks(path.normalize(path.join(projectWithCustomConfigurationFilePath, '.git', 'hooks')))
-    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`#!/bin/sh\nexit 1`, 'pre-push':`#!/bin/sh\nexit 1`}))
-
-    removeGitHooksFolder(projectWithCustomConfigurationFilePath)
+    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({ 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT }))
+    expect(isEqual(installedHooks, { 'pre-commit': TEST_SCRIPT, 'pre-push': TEST_SCRIPT })).toBe(true);
 })
+
+describe('Tests for skipping git hooks using SKIP_SIMPLE_GIT_HOOKS env var', () => {
+
+    const GIT_USER_NAME = "github-actions";
```

Comment:
I think we can do this in GitHub actions itself, do not see why we need some GH actions specific code in repo